lib/ui/screens/event_screen.dart

```diff
@@ -16,10 +16,11 @@ import 'package:url_launcher/url_launcher.dart';
 import 'package:reaxit/config.dart' as config;
 
 class EventScreen extends StatefulWidget {
-  final int pk;
+  final String? slug;
   final Event? event;
+  final int? pk;
 
-  EventScreen({required this.pk, this.event}) : super(key: ValueKey(pk));
+  const EventScreen({required this.pk, required this.slug, this.event});
```

Comment:
This needs an assert that not both slug and pk can be null. The `required` is also no longer neccesary